keyboards/lendunistus/rpneko65/rev1/info.json

```
@@ -0,0 +1,756 @@
+{
+    "manufacturer": "lendunistus",
+    "keyboard_name": "RPNeko65",
+    "maintainer": "lendunistus",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "matrix_pins": {
+        "cols": ["GP29", "GP15", "GP14", "GP13", "GP12", "GP11", "GP10", "GP9", "GP8", "GP6", "GP5", "GP4", "GP3", "GP2", "GP0", "GP1"],
+        "rows": ["GP28", "GP27", "GP26", "GP25", "GP24"]
+    },
+    "processor": "RP2040",
+    "rgblight": {
+        "led_count": 1,
```

remove erronoeus comma
```suggestion
        "led_count": 1
```